src/pages/gateway/getting-started.md

```diff
@@ -5,64 +5,30 @@ description: Set up and configure API Mesh for Adobe Developer App Builder.
 
 # Getting started
 
-This guide provides you with the basic steps you need to set up your API management environment:
-
-## Join the Beta
-
-Request access to join the API Mesh for Adobe Developer App Builder Beta:
-
-<!-- coming soon -->
-
--  An Adobe IO account. If you do not already have an Adobe IO account, [install Adobe IO].
+This guide provides you with the basic steps you need to set up your API management environment.
 
 ## Prerequisites
 
-Before you can begin using API Mesh for Adobe Developer App Builder, you must install:
+Before you can begin using the API Mesh, you must install:
 
 -  [Node.js]
 -  [nvm] 16.x.x
 
-You will also need to have:
+You will also need:
 
+-  An Adobe IO account. If you do not already have an Adobe IO account, [install Adobe IO].
```

Comment:
Technically, I don't think an instance of Commerce is required. I would list it as optional and mention that the examples refer to a Commerce instance.